screens/UserProfilePreview.screen.jsx

```diff
@@ -31,10 +49,34 @@ export const UserProfilePreviewScreen = () => {
     <FDefaultLayout>
       <>
         {!user ? <FSpinner /> : (
-          <FUserProfileCard
-            user={user}
-            isMe={false}
-          />
+          <>
+            <FUserProfileCard
+              user={user}
+              isMe={false}
+            />
+            <SafeAreaView>
+              <AnnouncementsList
+                horizontal
+                isMe={false}
+                userId={user.id}
+                pullData={getUserAnnouncementsLength}
+              />
+            </SafeAreaView>
+            {userAnnouncementsLength > 4 ? (
```

Comment:
zamiast dawać nulla to zrób po prostu userAnnouncementsLength > 4 && ten komponent